File: help/eng_gb/exercicios_da_ficha_3/Ficha03_018/Ficha03_018.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
#define length( a ) ( sizeof( a ) / sizeof( a[0] ))
//Main Function
int main() {
    printf("Positive number\t: ");
    int num;
    scanf("%d", &num);
    printf("Number of digits \t= %d",length( num + "" ) );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:58

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_018/Ficha03_018.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Positive number    : " ;
    $num = intval(isset($_POST["num"])? $_POST["num"] : isset($_GET["num"]) ? $_GET["num"] : 0);
    
    echo $num . "<br/>";// print input 
    echo  "Number of digits     = " . ( sizeof( $num . "" )  ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Tue, 2022 Sep 13 21:58

?>